evm/its.js

```diff
@@ -87,6 +87,24 @@ function compare(contractValue, configValue, variableName) {
     }
 }
 
+function isValidDestinationChain(config, chain, destinationChain, originalChain = null) {
```

Comment:
where is original chain passed? make this a generic util instead? can allow us to check if any chain is valid

Reply:
Added a more generic function to general utils to check if a chain exists in the config